services/scan/src/store.ts

```diff
@@ -299,21 +365,27 @@ export class Store {
     );
   }
 
-  getBallotPaperSizeForElection(): BallotPaperSize {
-    const electionDefinition = this.getElectionDefinition();
-    return (
-      electionDefinition?.election.ballotLayout?.paperSize ??
-      BallotPaperSize.Letter
-    );
-  }
-
   /**
    * Sets the current override values for mark thresholds. A value of undefined
    * will remove overrides and cause thresholds to fallback to the default values
    * in the election definition.
    */
   setMarkThresholdOverrides(markThresholds?: MarkThresholds): void {
-    this.setConfig(ConfigKey.MarkThresholdOverrides, markThresholds);
+    if (this.hasElection()) {
```

Comment:
Maybe short-circuit on `!this.hasElection()` to avoid indenting the whole thing?